### PR TITLE
Homework day3/day4

### PR DESCRIPTION
**Pull request by Zelderon7**

### CODE REVIEW

File: src/main/java/training_day3/HW/Zad1HW.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+public class Zad1HW {
+
+    public static void main(String[] args){
+
+    }
+
+    private static int Smallest(int a, int b, int c){
+        if(a < b)
+            return a < c? a:c;
```
Comment (Evgeni-Georgiev) [inline src/main/java/training_day3/HW/Zad1HW.java:11]: The ternary operator ca be formatted as so: a < c ? a : c.

File: src/main/java/training_day3/HW/Zad1HW.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        else
+            return b < c? b:c;
```
Comment (Evgeni-Georgiev) [inline src/main/java/training_day3/HW/Zad1HW.java:13]: The ternary operator ca be formatted as so: b < c ? b : c.

File: src/main/java/training_day3/HW/Zad2HW.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package training_day3.HW;
+
+public class Zad2HW {
+    public static void main(String[] args){
+
+    }
+
+    private static boolean isItEven(int a){
```
Comment (Evgeni-Georgiev) [inline src/main/java/training_day3/HW/Zad2HW.java:8]: "isEven" might be better.

**Review by Evgeni-Georgiev — CHANGES_REQUESTED** [review]
Good job.

File: src/main/java/training_day3/HW/Zad1HW.java (hunk 1 above)
Comment (Zelderon7) [reply src/main/java/training_day3/HW/Zad1HW.java:11]: tova za spac-ovete li samo stava duma?

Comment (Evgeni-Georgiev) [reply src/main/java/training_day3/HW/Zad1HW.java:11]: Please, write on english. And yes.

**Pull request merged**